site/profile/manifests/windows/baseline.pp

```diff
@@ -1,14 +1,13 @@
-class profile::windows::baseline {
+class profile::baseline {
```

Comment:
Can't rename is class to this as it has the wrong folder structure.  You should either move this file or create a new file called `site/profile/manifests/baseline.pp`